packages/code-studio/src/redux/selectors.ts

```diff
@@ -10,3 +10,13 @@ import LayoutStorage from '../storage/LayoutStorage';
 // eslint-disable-next-line import/prefer-default-export
 export const getLayoutStorage = (store: RootState): LayoutStorage =>
   store.layoutStorage as LayoutStorage;
+
+/**
+ * Get the configuration values of the server
+ * @param store The redux store
+ * @returns The layout storage instance
+ */
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
You can delete the empty line and the disabling of the warning, including for the export above as well. It's only necessary when the file only has one export, and we want it named.
```suggestion
```